tempesta_fw/http_msg.c

```diff
@@ -483,7 +483,7 @@ __hdr_sub(TfwHttpMsg *hm, char *name, size_t n_len, char *val, size_t v_len,
 
 	if (!TFW_STR_DUP(orig_hdr) && hdr.len <= orig_hdr->len) {
```

Comment:
The note https://github.com/natsys/tempesta/issues/444#issuecomment-205768369 is still not fixed